class-izooto-integration.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Izooto integration with PWA plugin.
+ */
+class Izooto_Integration {
+
+	public function __construct() {
+
+		add_action( 'wp_front_service_worker', [ $this, 'register_izooto_service_worker_script' ] );
+
+		// Remove izooto service worker so it does not conflict with PWA plugin service worker.
+		add_action(
+			'init',
+			function() {
+				remove_action( 'parse_request', 'izooto_sdk_files' );
+			}
+		);
+
+	}
+
+	/**
+	 * Register Izooto service workers scripts.
+	 *
+	 * @param object $scripts scripts object
+	 *
+	 * @return void
+	 */
+	public function register_izooto_service_worker_script( $scripts ) {
+
+		$scripts->register(
+			'izooto-workers', // Handle.
+			array(
+				'src' => function() {
+					require_once ABSPATH . 'wp-content/plugins/izooto-web-push/includes/class-init.php';
```

Comment:
Yes, it should be active. We can also add custom code snippets and remove the dependency if the features available in a plugin such as send notification on post publish or update are not required.

Reply:
If we add the custom code integration the notifications can only be sent by the izooto admin dashboard.

Reply:
@deepaklalwani97 Don't use hard coded path for `wp-content/plugins` folder because it can be change using constant `WP_PLUGIN_DIR` so use this constant.

`require_once WP_PLUGIN_DIR . '/izooto-web-push/includes/class-init.php';`